geometry_msgs/Point point
geometry_msgs/Point image_point